### ISSUE TITLE
Reproduce the result of paper

### ISSUE THREAD

**Issue by myutility** [Chrixtar/latentsplat#7]
Hi, 
First, thanks for the awesome work and code.
I tried to reproduce the results as baseline but had some confusion:
1. To train by myself and get the ckpt that has the same performance to your provided ckpt. Should I simply run python3 -m src.main  +experiment=co3d_hydrant with one gpu and take the last checkpoint(the one at 200000 iteration)?(The default bs is 2) Or are there any more details?
2. After I get the trained checkpoint, how can I get the statistics on the tables provided in your paper?
Thanks again!

**Comment by Chrixtar** [comment]
Hi,

you can either train the model yourself (option 1) or use one of the provided checkpoints directly for evaluation, skipping the training (option 2). Note that after training your model, you still need to run evaluation with your obtained checkpoint!
For option 1, training is explained [here](https://github.com/Chrixtar/latentsplat#training).
For option 2, evaluation with a given checkpoint path is explained [here](https://github.com/Chrixtar/latentsplat#evaluation).

What you will get in the end is a directory with reconstructed views of test scenes.
For computing the statistics, I used the `src/scripts/compute_metrics.py` script in combination with the evaluation config files in `config/evaluation`. Not that in the evaluation config file, e.g. `config/evaluation/co3d_hydrant_extra.yaml`, you need to specify the paths to the generated image directories from evaluation (you can comment out all other baseline methods).
The example for extrapolation on CO3D hydrants would look like this:
`python -m src.scripts.compute_metrics evaluation=co3d_hydrant_extra`

Feel free to reopen the issue in case of any further questions :)

Best
Chris

**Comment by myutility** [comment]
hi Chris,
Thanks for the detailed reply.
With your provided ckpt, I am able to reproduce the statistics of paper.
However, I also tried to train the model by myself.
Here is what I have done:
I ran on python3 -m src.main +experiment=co3d_hydrant on 2 gpus with config files unchanged. After training is done, I took last.ckpt and ran eval for hydrant extra:
And got: dist 0.266, lpips: 0.529, psnr: 13.855, ssim: 0.192. which is apparently not correct.
Is there anything wrong with my settings? Thanks!
